README.md

```diff
@@ -1,58 +1,86 @@
-- [ ] Links
+- [x] Links
+- [x] [Alternate Assets Object](https://github.com/stac-extensions/alternate-assets?tab=readme-ov-file#alternate-asset-object)
+
+| Field Name     | Type       | Description |
+| -------------- | ---------- | ----------- |
+| `storage:refs` | \[string\] | A property that specifies which schemes in `storage:schemes` may be used to access an Asset or Link. Each value must be one of the keys defined in `storage:schemes`. |
+
+### Storage Scheme Object
 
-| Field Name             | Type      | Description |
-| ---------------------- | --------- | ----------- |
-| storage:platform       | string    | The [cloud provider](#providers) where data is stored |
-| storage:region         | string    | The region where the data is stored. Relevant to speed of access and inter region egress costs (as defined by PaaS provider) |
-| storage:requester_pays | boolean   | Is the data requester pays or is it data manager/cloud provider pays. *Defaults to false* |
-| storage:tier           | string    | The title for the tier type (as defined by PaaS provider) |
+| Field Name     | Type    | Description |
+| -------------- | ------- | ----------- |
+| platform       | string  | **REQUIRED.** The [cloud provider](#platforms) where data is stored. |
+| region         | string  | The region where the data is stored. Relevant to speed of access and inter region egress costs (as defined by PaaS provider) |
+| requester_pays | boolean | Is the data requester pays or is it data manager/cloud provider pays. Defaults to `false` |
+| tier           | string  | The title for the tier type (as defined by PaaS provider) |
 
-While these are all valid properties on an Item, they will typically be defined per-asset. If a field applies equally
-to all assets (e.g., storage:platform=AWS if all assets are on AWS), then it should be specified in Item properties.
+The properties `title` and `description` as defined in Common Metadata can be used as well.
 
-### Additional Field Information
+#### Platforms
 
-#### Providers
-Currently this document is arranged to support object storage users of the following PaaS solutions:
+The `platform` field identifies the cloud provider where the data is stored.
+
+There are a couple of pre-defined values for common providers:
 
 - Alibaba Cloud (Aliyun): `ALIBABA`
 - Amazon AWS: `AWS`
 - Microsoft Azure: `AZURE`
 - Google Cloud Platform: `GCP`
 - IBM Cloud: `IBM`
 - Oracle Cloud: `ORACLE`
-- All other PaaS solutions: `OTHER`
 
-The upper-cased values are meant to be used for `storage:platform`.
+All other PaaS solutions must use a unique URL to the service.
+
+In case an `href` contains a non-HTTP URL that is not directly resolvable,
+the `platform` property must identify the host so that the URL can be resolved without further information.
+This is especially useful to provide the endpoint URL for custom S3 providers.
+In this case the `platform` is effectively the endpoint URL.
```

Comment:
I wasn't sure, thought it would give a smoother migration path. 
Open to change it if people think that's the way to go.
(Also I don't know which URLs apply to all the other providers.)

Reply:
I do like the platform being consistent and always a URL rather than having shortcuts which don't have specifically defined service URLs

Reply:
So what are the URLs for each of the predefined providers? I don't know it.

Reply:
I've just tried https://s3.amazonaws.com but that doesn't work for me. For AWS the URL should be the --endpoint-url that can be used in the AWS CLI for example. I just tried that for a bucket and the URL didn't work for me.

> In case an `href` contains a non-HTTP URL that is not directly resolvable,
> the `platform` property must identify the host so that the URL can be resolved without further information.
> This is especially useful to provide the endpoint URL for custom S3 providers.
> In this case the `platform` is effectively the endpoint URL.

Reply:
this could be related to the fact that AWS S3 doesn't do "request forwarding" (I think that's the name of it) for every region. For the "major" 6 or 7 regions, if you make a request to the wrong one and the bucket is actually in one of the other ones, S3 API will seamlessly forward you to the other one. So, when you use "s3.amazonaws.com" (an alias for us-east-1), if the bucket is actually in, say, us-west-2, you automatically get forwarded. If, however, it's in me-south-1, you get an error that the "location constraint" is wrong.

Reply:
By "identification", I meant exactly that, a definitive list of URI that identifies the platforms uniquely, regardless of how the final bucket/data/whatever is actually accessed. The idea is only to avoid sometimes using `"PLATFORM"` and sometimes a URI. In most cases,  the URI could be the corresponding doc (this is what `https://s3.amazonaws.com/` redirects to). It was never intended to be a "resolvable" URI to some specific bucket or "root" URI to those platforms to actually reach the contents, because that is impossible depending on the region always present in the endpoint URL, and which must be provided by `storage:schemes[].region`. 

Note that the same concept applies for the "other" platforms as well. It's not because a URI is provided that it will necessarily be resolvable. You might need some pre-auth operation or other condition/negotiation to access it.